pkg/oc/cli/secrets/options.go

```diff
@@ -49,7 +45,7 @@ func (o *SecretOptions) Complete(f kcmdutil.Factory, args []string) error {
 	if err != nil {
 		return err
 	}
-	kubeClient, err := kcoreclient.NewForConfig(clientConfig)
+	kubeClient, err := kcoreclientv1.NewForConfig(clientConfig)
```

Comment:
Assign directly to `o.KubeCoreClient`. Btw, you can just name that var `KubeClient`, that's shorter :wink: